.github/workflows/build.yml

```diff
@@ -246,7 +246,10 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: "16"
-      - run: cd clients/js && make test
+      - run: |
+          make docs
```

Comment:
Is this running `make` in the correct folder? I think this might always pass because it's not `cd`'ed into the clients/js folder, but maybe I'm missing something.

Reply:
good catch!